app/src/main/java/com/omkarmoghe/pokemap/controllers/net/NianticManager.java

```diff
@@ -332,7 +334,7 @@ public void getCatchablePokemon(final double lat, final double longitude, final
             public void run() {
                 try {
                     if (mPokemonGo != null && NianticManager.this.currentBatchCall == myCurrentBatch) {
-                        Thread.sleep(133);
+                        Thread.sleep(400);
```

Comment:
Ah sorry I forgot to change that back. I had it at 400 just to make sure all the requests would go through while testing